app/views/admin/publications/_search_results_list.html.erb

```diff
@@ -0,0 +1,20 @@
+<%
+  # Clients of this partial may override the following variable:
+  publishables ||= []
+%>
+
+<%=
+  osu.action_list(
+    records: publishables,
+    list: {
+      headings: [ 'Type', 'ID', 'Authors', 'Copyright Holders' ],
```

Comment:
Hmm, playing with this - would it be too much to get the title in here, as well? Just to allow a simple check that the admin is about to operate on the correct exercises. Since the titles are all blank, this is completely useless. Hmm. Make the id links clickable, to be able to view/verify a few? Ahhh, never mind, this is probably good enough.

Reply:
Yeah title is pretty useless right now